src/backend/users/models.py

```diff
@@ -2,4 +2,46 @@
 from django.contrib.auth.models import User
 
 class UserProfile(models.Model):
+    class TravelMode(models.TextChoices):
+        CAR = "CAR", "car"
+        BUSTRAIN = "BT", "bus/train"
```

Comment:
```suggestion
        BUS_TRAIN = "BT", "bus/train"
```
I think this (and the others here) would be more readable with underscores added, though it's also fine the way it is if you don't want to have to change all the migrations.